cmd/composectl/cmd/ps.go

```diff
@@ -63,7 +67,7 @@ func init() {
 
 var psCmd = &cobra.Command{
 	Use:   "ps",
-	Short: "ps <ref> [<ref>]",
+	Short: "ps [<ref> [<ref>]]",
```

Comment:
This is a more standard way to document this kind of parameter, where 0..N `refs` can be passed
```suggestion
	Short: "ps [ref]...",
```

Reply:
Updated.